dname.h

```diff
@@ -182,7 +182,7 @@ dname_label(const dname_type *dname, uint8_t label)
  *
  * Pre: left != NULL && right != NULL
  */
```

Comment:
Maybe noting the type in the comment is nice.
```suggestion
 * left and right are dname_type*.
 */
```

Reply:
Fixed